packages/eas-cli/src/fetch.ts

```diff
@@ -8,6 +10,16 @@ export class RequestError extends Error {
   }
 }
 
+function createHttpAgent(): Agent | null {
+  const httpProxyUrl = process.env.http_proxy;
+  if (!httpProxyUrl) {
+    return null;
+  }
+  return createHttpsProxyAgent(httpProxyUrl);
+}
+
+export const maybeHttpAgent: Agent | null = createHttpAgent();
```

Comment:
Maybe this should be `httpProxy`? I can see that `fetch` expects this to be called "agent" but I think it'd be better to treat it as a `fetch` "thing".

Reply:
this is http agent, the fact that it is changing the request to connect to proxy is not relevant to HTTP clients